lib/artirix_data_models/cached_action_adaptor.rb

```diff
@@ -44,10 +44,10 @@ def enabled?
   private
 
   def get_cached_result
-    return nil unless enabled?
+    raise "Unexpected NilClass returned from cache - case 2" unless enabled?
```

Comment:
actually this will never be called. The only place we call `get_cached_result` is after a `cached?` call. That one returns early with false unless it is `enabled?`. So this will never be called.